projects/plugins/crm/includes/ZeroBSCRM.Edit.php

```diff
@@ -170,35 +170,57 @@ public function preChecks(){
           $is_malformed_obj = false;
 
           if (is_array($this->obj) && isset($this->obj['owner'])){
-            $objOwner = (int)$this->obj['owner'];
+				$obj_owner = (int) $this->obj['owner'];
+
+				// This check covers the case where the contact or company is the owner of the transaction or quote
+				if ( isset( $this->obj['contact'][0]['owner'] ) ) {
+					$trans_quote_owner = (int) $this->obj['contact'][0]['owner'];
+
+				} elseif ( isset( $this->obj['company'][0]['owner'] ) ) {
+					$trans_quote_owner = (int) $this->obj['company'][0]['owner'];
+				// phpcs:disable Generic.WhiteSpace.ScopeIndent.IncorrectExact,Generic.WhiteSpace.ScopeIndent.Incorrect -- this sniff is incorrectly reporting spacing issues.
```

Comment:
I couldn't find an obvious fix for this sniff issue. This is what I was seeing: 
<img width="792" alt="Screenshot 2024-03-07 at 07 48 19" src="https://github.com/Automattic/jetpack/assets/16754605/dea66f56-31be-48c8-86ce-2847cd6279f5">

Reply:
Sometimes this old code makes the linter go haywire, I see no problem in disabling the error for this line.